src/rgw/rgw_sal_motr.cc

```diff
@@ -126,6 +126,13 @@ static std::string motr_global_indices[] = {
 #define TS_LEN 8
 #define UUID_LEN 23
 
+// log level mapping for RGW SAL
+#define LOG_CRITICAL 0
+#define LOG_ERROR 0
+#define LOG_WARNING 5
+#define LOG_INFO 10
+#define LOG_DEBUG 20
```

Comment:
I still feel that we don't need to use macros, putting a simple number with logs should be enough rather than these huge strings. 
It would be like HTTP requests where 404 represent not found or 200 represents success. 
We can always mention the mapping in comments at the top or in documentation and in logs we can use the corresponding integer values.

Reply:
I do agree macros will make future changes easier as we will need to make changes at only one place. But I don't see log level as something in which we will be making frequent changes so we can decide now and stick to it to avoid future changes.

Reply:
In HTTP  world, mapping of response code numbers to its meaning is well established (its a protocol, it has to be), and in RGW its not.
Point is, code is more readable and we people tend to follow the convention more when we have mapping intervened through these macros. 
Further reasoning is [here](https://github.com/Seagate/cortx-rgw/pull/404#discussion_r955698029)

Reply:
I am okay with this, in future when we have to make changes, this will make that process a lot quicker.